modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -273,13 +274,42 @@ def length(self):
         int
             The length of the object.
         """
+        self.try_build_length_cache()
+        return self._length_cache
```

Comment:
We need to unwrap `_length_cache` here, since its type will be `PandasDataframePartition`

Reply:
What do you mean by `unwrap`? Also, as far as I can tell, the logic for this method should be the same as it originally was (the code was just moved into the `try_build_length_cache`, so does this mean the original code returned `PandasDataframePartition` as well?